aio/deploy/helm-chart/kubernetes-dashboard/values.yaml

```
@@ -0,0 +1,174 @@
+# Default values for kubernetes-dashboard
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+image:
+  repository: kubernetesui/dashboard
+  tag: v2.0.0-beta8
+  pullPolicy: IfNotPresent
+  pullSecrets: []
+
+replicaCount: 1
+
+## Here annotations can be added to the kubernetes dashboard deployment
+annotations: {}
+## Here labels can be added to the kubernetes dashboard deployment
+labels: {}
+
+## Serve application over HTTP without TLS
+##
+## Note: If set to true, you may want to add --enable-insecure-login to extraArgs
+protocolHTTP: false
```

Casing of this default value does not match what is currently used throughout the templates.
```suggestion
protocolHttp: false
```

---

Note: I discovered this because I copied it from this file, and it wasn't working as expected until I changed my `--set` to use the properly cased value name.

---

Indeed, thanks for noticing, let me fix it and squash it.